### DISCUSSION TITLE
DrawTextureTiled rotation issue / question

### DISCUSSION THREAD

**Discussion by javierecf** [raysan5/raylib#4258] [Q&A] [answered]
Hello, I'm using the version of DrawTextureTiled from the examples, but when an angle is used I would like the individual texture calls to be rotating around the center of the destination size as if they were a single big texture.

Right now the behavior makes it as every call rotates around an origin but If I use the center of the destination size they don't keep in the same position

Angle: 0
![tiled1](https://github.com/user-attachments/assets/bef85c76-00e1-47b5-9922-535f2803d8cf)

Angle: 0.5
![tiled2](https://github.com/user-attachments/assets/d888e1dd-4e51-4e25-9ed0-92fc60164031)

does anyone know a way I can achieve that?, basically trying to do a tiled rotated rectangle around its center.

**Comment by JupiterRider** [comment, marked as answer]
Hey @javierecf ,
what about using a Camera2D (has a Rotation field) or a RenderTexture (you can draw the whole map on that texture and draw that texture rotated afterwards).

**Comment by javierecf** [comment]
@JupiterRider those are valid options I did not approached it like that, it may be worth the try, I'm not using Raylib anymore but if anyone else comes around this issue your answer may help them, thank you!
